src/assets/admin/js/alma.js

```diff
@@ -0,0 +1,243 @@
+jQuery(document).ready(function () {
+
+    almaAdminInternationalization();
+
+    var almaAdminGeneralHelper = new AlmaAdminHelper();
+    var almaAdminFeePlan = new AlmaAdminFeePlan(almaAdminGeneralHelper);
+    almaAdminFeePlan.renderFeePlan();
+    almaAdminFeePlan.initiateAlmaSelectMenuBehaviour();
+    almaAdminFeePlan.listenFeePlanCheckboxStatus();
+    almaAdminFeePlan.checkInputsOnSubmitActionTriggered();
+    almaAdminGeneralHelper.toggleTechnicalConfigFields();
+
+});
+
+/**
+ * Alma admin General Helpers.
+ */
+function AlmaAdminHelper() {
```

Comment:
Split this file into 3 files:
- alma.js
- helpers/AlmaAdminHelper.js
- helpers/AlmaAdminFeePlanHelper.js

Reply:
Noté pour plus tard !